_posts/2019-06-25-lambda-quickstart.md

```diff
@@ -0,0 +1,320 @@
+
+
+##### 2.5.2 Deploy chart
+
+Below are the commands to clone and deploy the official OpenFaaS Helm chart, configured to deploy to AWS Lambda.
+Please change the `--set` parameters `faaslambda.aws_region`, `faaslambda.lambda_execution_role`, `faaslambda.access_token` and
+`faaslambda.access_email` with the values you gathered together in the previous step.
+
+```bash
+$ git clone git@github.com:openfaas/faas-netes.git
+cd faas-netes
+helm repo add openfaas https://openfaas.github.io/faas-netes/
+helm repo update \
+ && helm upgrade openfaas --install openfaas/openfaas \
+                                     --namespace openfaas  \
+                                     --set faaslambda.create=true \
+                                     --set functionNamespace=openfaas-fn \
+                                     --set faaslambda.aws_region=eu-west-1 \
+                                     --set faaslambda.lambda_execution_role=arn:aws:iam::123456789012:role/openfaas-function-role \
+                                     --set faaslambda.access_token=eyJhbGciOiJFUzI1NiIsImtpZCI6IktrVnV5a2VDMkJCenRUREVsSzh5XzNzNlludlBiTHAtRmQ1djY2TGxvUU0iLCJ0eXAiOiJKV1QifQ.eyJhdWQiOiJjb29sZ3V5QGdtYWlsLmNvbSIsImV4cCI6MTU2ODU0NDQyMywiaWF0IjoxNTYwNTk1ODMwLCJpc3MiOiJmYWFzLWxhbWJkYSIsIm5iZiI6MTU2MDU5NTgzMCwic3ViIjoiZWFybHktYWNjZXNzIn0.0cKLO7nkjGQYwjNaQY2OLJFLYpz0bHxB-9btAU9TRrblOO8-hpz1pDRSajwg_SiY0IVRVZfaJRLFqeVy4FS8dA \
+                                     --set faaslambda.access_email=fredjones@gmail.com
+```
+
+### 3. Verifying the installation
+
+#### 3.1 Main OpenFaaS components are running
+
+```bash
+$ kubectl get pods -n openfaas
+
+kubectl get pods -n openfaas                                                                 290ms  Wed 26 Jun 08:06:38 2019
+NAME                            READY   STATUS    RESTARTS   AGE
+alertmanager-775d5cd49f-2sf2b   1/1     Running   0          8d
+faas-idler-57b849dfd-k9rfv      1/1     Running   0          8d
+gateway-59d9dbbf-6znm2          2/2     Running   0          8d
+nats-75d8f56846-j2b4b           1/1     Running   0          8d
+prometheus-5945df7857-rt2fw     1/1     Running   0          8d
+queue-worker-5f96b567c5-pnsgd   1/1     Running   0          8d
+
+$ faas-cli version
+
+  ___                   _____           ____
+ / _ \ _ __   ___ _ __ |  ___|_ _  __ _/ ___|
+| | | | '_ \ / _ \ '_ \| |_ / _` |/ _` \___ \
+| |_| | |_) |  __/ | | |  _| (_| | (_| |___) |
+ \___/| .__/ \___|_| |_|_|  \__,_|\__,_|____/
+      |_|
+
+CLI:
+ commit:  25cada08609e00bed526790a6bdd19e49ca9aa63
+ version: 0.8.14
+
+Gateway
+ uri:     http://localhost:8083
+
+Provider
+ name:          faas-lambda
+ orchestration: lambda
+ version:       dev
+ sha:           6de5591299665e5cd0b984b7e0ef195dd3e2d2ea
+```
+
+#### 3.2 Deploy your first OpenFaaS function to AWS Lambda
+
+**Deploy function**
+```bash
+$ faas store deploy "SSL/TLS cert info"
+
+
+Deployed. 200 OK.
+URL: http://localhost:8083/function/certinfo
```

Comment:
Where did `localhost:8083` come from? Shouldn't this be the LoadBalancer IP from Kubernetes?